package.json

```diff
@@ -17,6 +17,11 @@
       "type": "string",
       "default": "ruby"
     },
+    "rubyExecutableArgs": {
+      "title": "Arguments passed to Ruby",
+      "type": "string",
+      "default": "-c -w --external-encoding=utf-8 --internal-encoding=utf-8"
```

Comment:
This should be _additional_ arguments to pass to Ruby, these ones _must_ be there for the provider to work so there is no reason to let the user remove them.

Reply:
Done in 5ccf337532dc7d1c261b83c683c6ecc3277b54b3 🙂